packages/node/src/meta/meta.module.ts

```diff
@@ -57,8 +61,20 @@ import { MetaService } from './meta.service';
       name: 'subql_indexer_skip_dictionary_count',
       help: 'The number of times indexer been skip use dictionary',
     }),
+    {
+      provide: MetaService,
+      useFactory: async (
+        indexerManager: IndexerManager,
+        storeService: StoreService,
+      ) => {
+        await indexerManager.start();
```

Comment:
I need to find a better way to do this. Because currently indexerManager.start() is being called twice.